src/planscape/conditions/management/commands/create_tiles.py

```diff
@@ -70,6 +70,10 @@ def _convert_tif_to_tiles(self, dry_run: bool, data_directory: str, filepath: st
         gdal2tiles = ['gdal2tiles.py', '-s', settings.CRS_9822_PROJ4,
                       '--xyz', '-z', zoom_levels, gdaldem_file, output_file]
         self.stdout.write('-----------------------')
+        self.stdout.write(output_file)
+        if os.path.exists(output_file):
+            self.stdout.write('SKIPPING: ' + output_file)
```

Comment:
does this skip saving the new tiles if the file path already exists? at some point, we'll want to overwrite them to incorporate the newly projected rasters

Reply:
Let me fix that.  I'll add another flag (force!)